client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,1316 @@
+								Explore our Professional plans
+							</button>
+						</>
+					) }
+				</FreeBanner>
+				<SharedFeatures selectedTab={ selectedTab } sharedFeatures={ sharedFeatures } />
+			</Grid>
+		</>
+	);
+}
+
+TabbedPlans.propTypes = {
+	onUpgradeClick: PropTypes.func,
+};
+
+const mapStateToProps = ( state, ownProps ) => {
+	const { plans } = ownProps;
+	// eslint-disable-next-line wpcalypso/redux-no-bound-selectors
+	const planProperties = plans.map( ( plan ) => {
+		const planConstantObj = applyTestFiltersToPlansList( plan, undefined );
+		const planProductId = planConstantObj.getProductId();
+		const planObject = getPlan( state, planProductId );
+		const rawPrice = getPlanRawPrice( state, planProductId, true );
+		const isMonthlyPlan = isMonthly( plan );
+		let annualPricePerMonth = rawPrice;
+		if ( isMonthlyPlan ) {
+			// Get annual price per month for comparison
+			const yearlyPlan = getPlanBySlug( state, getYearlyPlanByMonthly( plan ) );
+			if ( yearlyPlan ) {
+				annualPricePerMonth = getPlanRawPrice( state, yearlyPlan.product_id, true );
+			}
+		}
+		const monthlyPlanKey = findPlansKeys( {
+			group: planConstantObj.group,
+			term: TERM_MONTHLY,
+			type: planConstantObj.type,
+		} )[ 0 ];
+		const monthlyPlanProductId = getPlanFromKey( monthlyPlanKey )?.getProductId();
+		// This is the per month price of a monthly plan. E.g. $14 for Premium monthly.
+		const rawPriceForMonthly = getPlanRawPrice( state, monthlyPlanProductId, true );
+
+		return {
+			annualPricePerMonth,
+			currencyCode: getCurrentUserCurrencyCode( state ),
+			planName: planObject?.product_name_short,
+			planObject,
+			planProductId,
+			planSlug: planObject?.product_slug,
+			rawPrice,
+			rawPriceForMonthly,
+			termLength: isMonthlyPlan ? 'monthly' : 'annually',
+		};
+	} );
+	planProperties.unshift( {
+		annualPricePerMonth: 0,
+		currencyCode: getCurrentUserCurrencyCode( state ),
+		planName: 'Free',
+		planObject: {},
+		planProductId: 1,
+		planSlug: 'free_plan',
+		rawPrice: 0,
+		rawPriceForMonthly: 0,
+		termLength: null,
+	} );
+	return {
+		planProperties,
+	};
+};
+
+export default connect( mapStateToProps )( TabbedPlans );
+
+function getTabList() {
+	return [ 'Starter', 'Professional' ];
+}
+
+function getBestForStrings() {
+	return {
+		Free: 'Ads displayed on all sites',
+		Personal: 'Best for small sites & blogs',
+		Premium: 'Best for customized sites & blogs',
+		Business: 'Best for professional sites & blogs',
+		eCommerce: 'Best for online stores',
+	};
+}
+
+function getFeatureComparisonData() {
+	const data = {
+		Professional: [
+			{
+				featureName: 'Install plugins',
+				tooltip:
+					'Plugins are like apps for your site. They add new functionality and features to expand your site. There are over 58,000+ WordPress plugins available for anything you need.',
+				planOne: {
+					annually: {
+						included: false,
+						copy: <Gridicon icon="cross" size={ 18 } />,
+						mobileCopy: null,
```

Comment:
*This is quite opinionated so please feel free to ignore*

There are many cases that `mobileCopy` is actually the same as `copy`. Thus, I'd prefer the other way around: `mobileCopy` is used for overriding. i.e. it will only be filled in when there is a deviation of copy between desktop and mobile. For cases that it needs to be hidden, I'd prefer an explicit `excludeOnMobile` prop instead of reusing the same prop.

Again, this is quite a personal preference thing, so please feel free to keep it your way :)